client/src/app/project-registration/project-id/project-id.component.ts

```diff
@@ -35,7 +40,7 @@ export class ProjectIdComponent implements OnInit {
 
   technology: string;
   otherTechnology: string;
-  contributor: string;
+  contributor_name: string;
```

Comment:
camelCase?

Reply:
that was part of #54 that was already merged, brought in so I could resolve merge conflicts.